webcomponents/input/input.webcomponent.spec.js

```diff
@@ -176,11 +176,30 @@ describe('mn-input (webcomponent)', () => {
     })
 
     it('should set empty text to label when attribute is removed', () => {
-      element.setAttribute('placeholder','test')
+      element.setAttribute('placeholder', 'test')
       element.removeAttribute('placeholder')
```

Comment:
after to run this line, the attribute **is not being excluded**, as show below:

![max_length](https://cloud.githubusercontent.com/assets/23660750/26319344/081b766c-3ef5-11e7-8a3f-f67cc7db95e6.png)

And even existing, the test is passing.